apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/ConcertCoverPic.test.tsx

```
@@ -0,0 +1,315 @@
+        it('renders the main container with correct dimensions and styling', () => {
+            const container = document.querySelector('.w-full.h-\\[250px\\]');
+            expect(container).toBeInTheDocument();
+            expect(container).toHaveClass('relative', 'overflow-hidden');
+        });
+
+        it('renders the background image', () => {
+            const image = screen.getByRole('img');
+            expect(image).toBeInTheDocument();
+            expect(image).toHaveAttribute(
+                'src',
+                'https://wallpapers.com/images/featured-full/funny-baby-pictures-0wuld2ho84vak3lp.jpg'
```

ESLint

[error] Found a string with entropy 4.08 : "https://wallpapers.com/images/featured-full/funny-baby-pictures-0wuld2ho84vak3lp.jpg"

(no-secrets/no-secrets)